aws_batch/pipeline.py

```diff
@@ -784,6 +784,23 @@ def run_sample(sample_s3_input_path, sample_s3_output_path,
         result_dir + '/' + UNIDENTIFIED_FASTA_OUT,
         result_dir, sample_s3_output_path, False)
 
+def run_star_part(output_dir, genome_dir, fastq_file_1, fastq_file_2):
+    star_command_params = ['cd', output_dir, ';', STAR,
+                           '--outFilterMultimapNmax', '99999',
+                           '--outFilterScoreMinOverLread', '0.5',
+                           '--outFilterMatchNminOverLread', '0.5',
+                           '--outReadsUnmapped', 'Fastx',
+                           '--outFilterMismatchNmax', '999',
+                           '--outSAMmode', 'None',
+                           '--clip3pNbases', '0',
+                           '--runThreadN', str(multiprocessing.cpu_count()),
+                           '--genomeDir', genome_dir,
+                           '--readFilesIn', fastq_file_1, fastq_file_2]
+    if fastq_file_1[-3:] == '.gz':
+        ['--readFilesCommand', 'zcat']
```

Comment:
star_command_params.extend(['--readFilesCommand', 'zcat']) ?

Reply:
good catch. meant to do +=